### PR TITLE
Change vegan status of 'lactic ferments' and 'fat'

### PR DESCRIPTION
**Pull request by blazern**
Changing vegan status of 'lactic ferments' and 'fat'.

Motivation for `fat`: almost every product contains fat, for example sunflower oil is 99% fat, but also is vegan.
Motivation for `lactic ferments` is written into the comment in the file.

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -16406,7 +16406,7 @@ zh:脂肪
 wikidata:en:Q127980
 wikipedia:en:https://en.wikipedia.org/wiki/Fat
 # ingredient/fat has 7494 products in 29 languages @2019-07-10
-vegan:en:no
+vegan:en:maybe
 vegetarian:en:maybe
 from_palm_oil:en:no
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:16411]: not related to your change, but I guess from_palm_oil:en:no should be from_palm_oil:en:maybe

**Review by stephanegigandet — APPROVED** [review]
Looks good to me.

**Comment by stephanegigandet** [comment]
There's one failing test:

#   Failed test at t/ingredients_analysis.t line 59.
#     Structures begin differing at:
#          $got->[1] = 'en:maybe-vegan'
#     $expected->[1] = 'en:non-vegan'

It should be updated in t/ingredients_analysis.t which currently contains this:

[ { lc => "en", ingredients_text => "fat, proteins" }, [ "en:palm-oil-free", "en:non-vegan", "en:maybe-vegetarian"] ],

@blazern : could you update the test, and maybe add one with ingredients containing lactic ferments?

**Comment by blazern** [comment]
Another test started failing and I cannot understand how it can be fixed.
@stephanegigandet, could you help me a bit please?

**Comment by stephanegigandet** [comment]
@blazern : I sent you an invite to the openfoodfacts-server team on github, that should allow you to push to branches that are directly on the openfoodacts-server repo. Could you push your changes on a branch there? That way I will be able to make changes to it, to make the tests pass. Thanks!

**Comment by blazern** [comment]
Opened a clone PR: https://github.com/openfoodfacts/openfoodfacts-server/pull/4970

**Pull request closed without merging**